README.md

```diff
@@ -372,6 +376,14 @@ the *project* parameter in *integration-test-config.json* (or elsewhere).
 Notice that the file name must be exactly *integration-test-config.json* as that is what the
 integration test will search for when it is run on the maven verify phase.
 
+#### Integration testing with application default credentials
+
+Due to a limitation in the
+[GCS Connector library](https://cloud.google.com/dataproc/docs/concepts/connectors/cloud-storage),
+the integration tests do not work when using  application default credentials, unless the tests are launched on
```

Comment:
There's a double space between between the two words here `using  application`, other than that. Great!